testcases/MachineConfig.py

```diff
@@ -191,19 +218,16 @@ def runTest(self):
             try: self.pmem_size = conf.args.pmem_size
             except AttributeError:
                 self.pmem_size = "8192"
+            self.cv_HMC.remove_vpmem()
```

Comment:
a simple check to know if already vpmem exists and than remove, instead always remove,, most of the lpars will not have vpmem or fresh vpmem create

Reply:
yes good idea will do